conftest.py

```diff
@@ -47,13 +47,13 @@ def default_localstack_client_fixture() -> Iterator[None]:
     This is required, because some tests explicelty check the returned arn, and by default LocalStack will assume ID 00000000
     """
 
-    def localstack_mock_env_variables(self, org=BaseMockAWS.mock_env_variables):
+    def localstack_mock_env_variables(self, org=MockAWS._mock_env_variables):
         # patch the access key id to use the same one in LocalStack
         self.FAKE_KEYS["AWS_ACCESS_KEY_ID"] = DEFAULT_ACCOUNT_ID
```

Comment:
i think this may be the (or at least one) root cause for failing tests. In the logs I see a few errors
```
 AttributeError: 'ServerModeMockAWS' object has no attribute 'FAKE_KEYS'
```
I could followup on that as well @alexrashed

Reply:
Thanks for digging into this! I pushed a fixed-up commit with this fix.
I'm happy if you take it from here if there's something else.